cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -118,7 +118,9 @@ protected void createContent(final Bundle savedInstanceState) {
             // autoplay video with url
             "--autoplay-policy=no-user-gesture-required",
             // remove below if Cobalt rebase to m120+
-            "--user-level-memory-pressure-signal-params"
+            "--user-level-memory-pressure-signal-params",
+            // pass javascript console log to adb log
```

Comment:
Nit: Comments should be full sentences, i.e. `Pass javascript console log to adb log.` (Capitalize first, period at the end).

Reply:
the comment aligns with all the comment in this section, should I keep it as it is, or should I modify all of them?

Reply:
Since you asked, please modify all of them : )

You can do so in a follow-up.